doc/modules/model_evaluation.rst

```diff
@@ -1109,6 +1110,71 @@ function:
     >>> matthews_corrcoef(y_true, y_pred)  # doctest: +ELLIPSIS
     -0.33...
 
+.. _multilabel_confusion_matrix:
+
+Multi-label confusion matrix
+----------------
+
+The :func:`multilabel_confusion_matrix` function computes multilabel confusion
+matrix class-wisely (default) or sample-wisely (samplewise=True) to evaluate
```

Comment:
"-wise", not "-wise", please

Reply:
you mean not "-wisely"? that's a little weird to me. Well, if you insist...

Reply:
It's definitely more acceptable English.